webapp/sass/layout/_headers.scss

```diff
@@ -454,12 +454,20 @@
             text-decoration: none;
 
             &:hover {
-                @include opacity(1);
+                @include opacity(0.5);
```

Comment:
Can we increase this to (.7), I know we had a lower opacity in the docs, but I'll probably update that because looking at it, the hover opacity looks a bit low.